tools/cockpit.spec

```diff
@@ -419,7 +419,7 @@ This package contains the Cockpit shell and system configuration interfaces.
 %package tests
 Summary: Tests for Cockpit
 Requires: %{name}-bridge >= %{version}-%{release}
-Requires: %{name}-shell >= %{version}-%{release}
+Requires: %{name}-system >= %{version}-%{release}
```

Comment:
The debian -tests package currently doesn't have any dependencies to cockpit* at all, I figure it should grow a `Depends: cockpit-system (>= ${source:Version})` (we can't use `= ${binary:Version}` as c-system is Arch: all).